cfglib.py

```diff
@@ -443,6 +447,10 @@
 dst_identity_opts = [
     cfg.StrOpt('service', default='keystone',
                help='name service for keystone'),
+    cfg.BoolOpt('optimize_user_role_fetch', default=False,
```

Comment:
I'm not sure this option is required for destination cloud, are we reading user roles on destination too?..